(cd /export/home/drlou/illumos-userland-1248/components/python/python26/build/i86 ; /usr/bin/env  /usr/gnu/bin/make \
			 test)
(cd /export/home/drlou/illumos-userland-1248/components/python/python26/build/amd64 ; /usr/bin/env  /usr/gnu/bin/make \
			 test)
make[1]: Entering directory `/export/home/drlou/illumos-userland-1248/components/python/python26/build/i86'
make[1]: Entering directory `/export/home/drlou/illumos-userland-1248/components/python/python26/build/amd64'
running build
running build_ext
running build_ext
[?1034h
Failed to find the necessary bits to build these modules:
_bsddb             bsddb185           linuxaudiodev   
ossaudiodev                                           
To find the necessary bits, look in setup.py in detect_modules() for the module's name.

running build_scripts
find /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib -name '*.py[co]' -print | xargs rm -f
[?1034h
Failed to find the necessary bits to build these modules:
_bsddb             bsddb185           dl              
imageop            linuxaudiodev      ossaudiodev     
To find the necessary bits, look in setup.py in detect_modules() for the module's name.

running build_scripts
find /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib -name '*.py[co]' -print | xargs rm -f
LD_LIBRARY_PATH=/export/home/drlou/illumos-userland-1248/components/python/python26/build/amd64: ./python -E -tt /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/regrtest.py -l 
test_grammar
test_grammar
test_opcodes
test_opcodes
test_dict
test_dict
test_builtin
test_builtin
test_exceptions
test_exceptions
test_types
test_types
test_unittest
test_unittest
test_doctest
test_doctest
[?1034htest_doctest2
test_MimeWriter
[?1034htest_doctest2
test_MimeWriter
test_SimpleHTTPServer
test_SimpleHTTPServer
test_StringIO
test_StringIO
test___all__
test___all__
test___future__
test___future__
test__locale
test__locale
test_abc
test_abc
test_abstract_numbers
test_abstract_numbers
test_aepack
test_aepack
test_aepack skipped -- No module named aepack
test_aifc
test_aifc
test_al
test_al skipped -- No module named al
test_al
test_al skipped -- No module named al
test_anydbm
test_anydbm
test_applesingle
test_applesingle skipped -- No module named macostools
test_array
test_applesingle
test_applesingle skipped -- No module named macostools
test_array
test_ast
test_ast
test_asynchat
test_asynchat
test_asyncore
test_asyncore
test_atexit
test_atexit
test_audioop
test_audioop
test_augassign
test_augassign
test_base64
test_base64
test_bastion
test_bastion
test_bigaddrspace
test_bigaddrspace
test_bigmem
test_bigmem
test_binascii
test_binascii
test_binhex
test_binhex
test_binop
test_binop
test_bisect
test_bisect
test_bool
test_bool
test_bsddb
test_bsddb
test_bsddb skipped -- No module named _bsddb
test_bsddb skipped -- No module named _bsddb
test_bsddb185
test_bsddb185
test_bsddb185 skipped -- No module named bsddb185
test_bsddb3
test_bsddb3 skipped -- Use of the `bsddb' resource not enabled
test_bsddb3
test_bsddb3 skipped -- Use of the `bsddb' resource not enabled
test_buffer
test_buffer
test_bufio
test_bufio
test_bytes
test_bytes
test_bz2
test_bz2
test_calendar
test_call
test_calendar
test_capi
test_call
test_capi
test_cd
test_cd skipped -- No module named cd
test_cfgparser
test_cd
test_cd skipped -- No module named cd
test_cfgparser
test_cgi
test_cgi
test_charmapcodec
test_charmapcodec
test_cl
test_cl skipped -- No module named cl
test_cl
test_cl skipped -- No module named cl
test_class
test_class
test_cmath
test_cmath
test_cmd
test_cmd
test_cmd_line
test_cmd_line
test_cmd_line_script
test_cmd_line_script
test_code
test_code
test_codeccallbacks
test_codeccallbacks
test_codecencodings_cn
test_codecencodings_cn
test_codecencodings_hk
test_codecencodings_jp
test_codecencodings_hk
test_codecencodings_jp
test_codecencodings_kr
test_codecencodings_kr
test_codecencodings_tw
test_codecencodings_tw
test_codecmaps_cn
test_codecmaps_cn skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_hk
test_codecmaps_hk skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_jp
test_codecmaps_jp skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_kr
test_codecmaps_kr skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_cn
test_codecmaps_cn skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_tw
test_codecmaps_tw skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_hk
test_codecmaps_hk skipped -- Use of the `urlfetch' resource not enabled
test_codecs
test_codecmaps_jp
test_codecmaps_jp skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_kr
test_codecmaps_kr skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_tw
test_codecmaps_tw skipped -- Use of the `urlfetch' resource not enabled
test_codecs
test_codeop
test_codeop
test_coding
test_coding
test_coercion
test_coercion
test_collections
test_collections
test_colorsys
test_commands
test_colorsys
test_commands
test_compare
test_compare
test_compile
test_compile
test_compiler
test_compiler
test_complex
test_complex_args
test_contains
test_contextlib
test_cookie
test_cookielib
test_copy
test_copy_reg
test_cpickle
test_cprofile
test_crypt
test_csv
test_ctypes
test_complex
test test_ctypes failed -- errors occurred; run in verbose mode for details
test_curses
test_curses skipped -- Use of the `curses' resource not enabled
test_complex_args
test_datetime
test_contains
test_contextlib
test_cookie
test_cookielib
test_copy
test_copy_reg
test_cpickle
test_cprofile
test_crypt
test_csv
test_ctypes
test test_ctypes failed -- errors occurred; run in verbose mode for details
test_curses
test_curses skipped -- Use of the `curses' resource not enabled
test_datetime
test_dbm
test_decimal
test_dbm
test_decimal
test_decorators
test_defaultdict
test_deque
test_decorators
test_defaultdict
test_deque
test_descr
test_descr
test_descrtut
test_difflib
test_descrtut
test_difflib
test_dircache
test_dircache
test_dis
test_distutils
test_dis
test test_distutils failed -- errors occurred; run in verbose mode for details
test_dl
test_dl skipped -- No module named dl
test_docxmlrpc
test_distutils
test test_distutils failed -- errors occurred; run in verbose mode for details
test_dl
test_docxmlrpc
test_dumbdbm
test_dumbdbm
test_dummy_thread
test_dummy_threading
test_email
test_dummy_thread
test_dummy_threading
test_email
test_email_codecs
test_email_renamed
test_email_codecs
test_email_renamed
test_enumerate
test_enumerate
test_eof
test_epoll
test_epoll skipped -- test works only on Linux 2.6
test_eof
test_epoll
test_epoll skipped -- test works only on Linux 2.6
test_errno
test_errno
test_exception_variations
test_exception_variations
test_extcall
test_extcall
test_fcntl
test_fcntl
test_file
test_file
test_filecmp
test_fileinput
test_filecmp
test_fileio
test_fileinput
test_float
test_fileio
test_float
test_fnmatch
test_fnmatch
test_fork1
test_fork1
test_format
test_format
test_fpformat
test_fpformat
test_fractions
test_fractions
test_frozen
test_frozen
test_ftplib
test_ftplib
test_funcattrs
test_funcattrs
test_functools
test_functools
test_future
test_future
test_future3
test_future3
test_future4
test_future4
test_future5
test_future5
test_future_builtins
test_future_builtins
test_gc
test_gc
test_gdbm
test_gdbm
test_generators
test_genericpath
test_genexps
test_getargs
test_getargs2
test_getopt
test_gettext
test_gl
test_generators
test_gl skipped -- No module named gl
test_glob
test_global
test_genericpath
test_grp
test_genexps
test_gzip
test_getargs
test_getargs2
test_getopt
test_gettext
test_gl
test_gl skipped -- No module named gl
test_glob
test_hash
test_global
test_hashlib
test_grp
test_gzip
test_heapq
test_hash
test_hashlib
test_heapq
test_hmac
test_hotshot
test_htmllib
test_hmac
test_hotshot
test_htmllib
test_htmlparser
test_htmlparser
test_httplib
test_httplib
test_httpservers
test_httpservers
test_imageop
test_imageop
test_imageop skipped -- No module named imageop
test_imaplib
test_imaplib
test_imgfile
test_imgfile skipped -- No module named imgfile
test_imgfile
test_imgfile skipped -- No module named imgfile
test_imp
test_imp
test_import
test_import
test_importhooks
test_importhooks
test_index
test_index
test_inspect
test_inspect
test_int
test_int
test_int_literal
test_int_literal
test_io
test_io
test_ioctl
test_isinstance
test_iter
test_iterlen
test_itertools
test_ioctl
test_isinstance
test_iter
test_iterlen
test_itertools
test_json
test_json
test_kqueue
test_kqueue skipped -- test works only on BSD
test_largefile
test_lib2to3
test_kqueue
test_kqueue skipped -- test works only on BSD
test_largefile
test_lib2to3
test_linecache
test_linuxaudiodev
test_linuxaudiodev skipped -- Use of the `audio' resource not enabled
test_list
test_locale
test_logging
test_linecache
test_linuxaudiodev
test_linuxaudiodev skipped -- Use of the `audio' resource not enabled
test_list
test_locale
test_logging
test_long
test_long
test_long_future
test_longexp
test_macos
test_macos skipped -- No module named MacOS
test_macostools
test_macostools skipped -- No module named macostools
test_macpath
test_long_future
test_mailbox
test_longexp
test_macos
test_macos skipped -- No module named MacOS
test_macostools
test_macostools skipped -- No module named macostools
test_macpath
test_mailbox
test_marshal
test_math
test_marshal
test_md5
test_memoryio
test_math
test_mhlib
test_md5
test_memoryio
test_mhlib
test_mimetools
test_mimetypes
test_mimetools
test_minidom
test_mimetypes
test_minidom
test_mmap
test_mmap
test_module
test_module
test_modulefinder
test_modulefinder
test_multibytecodec
test_multibytecodec
test_multibytecodec_support
test_multifile
test_multiprocessing
test_multibytecodec_support
test_multifile
test_multiprocessing
test test_multiprocessing failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_multiprocessing.py", line 1209, in test_rapid_restart
    manager.start()
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/multiprocessing/managers.py", line 499, in start
    self._address = reader.recv()
EOFError

test_mutants
test_mutants
test_mutex
test_mutex
test_netrc
test_netrc
test_new
test_new
test_nis
test_nis
test_normalization
test_normalization skipped -- Use of the `urlfetch' resource not enabled
test_normalization
test_normalization skipped -- Use of the `urlfetch' resource not enabled
test_ntpath
test_ntpath
test_old_mailbox
test_old_mailbox
test_openpty
test_openpty
test_operator
test_operator
test_optparse
test_optparse
test_os
test_os
test_ossaudiodev
test_ossaudiodev
test_ossaudiodev skipped -- Use of the `audio' resource not enabled
test_ossaudiodev skipped -- Use of the `audio' resource not enabled
test_parser
test_parser
test_peepholer
test_peepholer
test_pep247
test_pep247
test_pep263
test_pep263
test test_pep263 crashed -- <type 'exceptions.SyntaxError'>: Non-ASCII character '\xf0' in file /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_pep263.py on line 11, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (test_pep263.py, line 11)
test test_pep263 crashed -- <type 'exceptions.SyntaxError'>: Non-ASCII character '\xf0' in file /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_pep263.py on line 11, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (test_pep263.py, line 11)
test_pep277
test_pep277 skipped -- test works only on NT+
test_pep277
test_pep277 skipped -- test works only on NT+
test_pep292
test_pep292
test_pep352
test_pep352
test_pickle
test_pickle
test_pickletools
test_pickletools
test_pipes
test_pkg
test_pipes
test_pkgimport
test_pkg
test_pkgutil
test_pkgimport
test_platform
test_pkgutil
test_platform
test_plistlib
test_plistlib
test_poll
test_poll
test_popen
test_popen
test_popen2
test_popen2
test_poplib
test_poplib
test_posix
test_posix
test test_posix failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 267, in test_getcwd_long_pathnames
    _create_and_do_getcwd(dirname)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 260, in _create_and_do_getcwd
    os.getcwd()
OSError: [Errno 34] Result too large

test test_posix failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 267, in test_getcwd_long_pathnames
    _create_and_do_getcwd(dirname)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 260, in _create_and_do_getcwd
    os.getcwd()
OSError: [Errno 34] Result too large

test_posixpath
test_posixpath
test_pow
test_pow
test_pprint
test_pprint
test_print
test_print
test_profile
test_profile
test_profilehooks
test_profilehooks
test_property
test_property
test_pstats
test_pstats
test_pty
test_pty
test_pwd
test_pwd
test_py3kwarn
test_py3kwarn skipped -- test.test_py3kwarn must be run with the -3 flag
test_py3kwarn
test_py3kwarn skipped -- test.test_py3kwarn must be run with the -3 flag
test_pyclbr
test_pyclbr
test_pydoc
test_pydoc
test_pyexpat
test_queue
test_pyexpat
test_queue
test_quopri
test_quopri
test_random
test_random
test_re
test_re
test_repr
test_repr
test_resource
test_rfc822
test_resource
test_rfc822
test_richcmp
test_richcmp
test_robotparser
test_robotparser
test_runpy
test_runpy
test_sax
test_sax
test_scope
test_scope
test_scriptpackages
test_scriptpackages skipped -- No module named aetools
test_scriptpackages
test_scriptpackages skipped -- No module named aetools
test_select
test_select
test_set
test_set
test_sets
test_sets
test_sgmllib
test_sgmllib
test_sha
test_sha
test_shelve
test_shelve
test_shlex
test_shutil
test_signal
test_shlex
test_shutil
test_signal
test_site
test_site
test_slice
test_slice
test_smtplib
test_smtplib
test_smtpnet
test_smtpnet skipped -- Use of the `network' resource not enabled
test_smtpnet
test_smtpnet skipped -- Use of the `network' resource not enabled
test_socket
test_socket
test_socketserver
test_socketserver
test_socketserver skipped -- Use of the `network' resource not enabled
test_socketserver skipped -- Use of the `network' resource not enabled
test_softspace
test_softspace
test_sort
test_sort
test_sqlite
test_sqlite
test_ssl
test_startfile
test_startfile skipped -- cannot import name startfile
test_ssl
test_str
test_startfile
test_startfile skipped -- cannot import name startfile
test_str
test_strftime
test_strftime
test_string
test_string
test_stringprep
test_strop
test_strptime
test_stringprep
test_strop
test_struct
test_strptime
test_struct
test_structmembers
test_structseq
test_subprocess
test_structmembers
test_structseq
test_subprocess
.
    this bit of output is from a test of stdout in a different process ...
.
    this bit of output is from a test of stdout in a different process ...
test_sunaudiodev
test_sunaudiodev skipped -- no audio device found!
test_sundry
test_symtable
test_syntax
test_sys
test test_sys failed -- errors occurred; run in verbose mode for details
test_tarfile
test_tcl
test_telnetlib
test_sunaudiodev
test_sunaudiodev skipped -- no audio device found!
test_sundry
test_symtable
test_syntax
test_sys
test test_sys failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_sys.py", line 359, in test_43581
    self.assert_(sys.__stdout__.encoding == sys.__stderr__.encoding)
AssertionError

test_tarfile
test_tempfile
test_textwrap
test_thread
test_threaded_import
test_threadedtempfile
test_tcl
test_threading
test_telnetlib
test_tempfile
test_textwrap
test_thread
test_threading_local
test_threaded_import
test_threadedtempfile
test_threading
test_threadsignals
test_time
test_threading_local
test_threadsignals
test_time
test_timeout
test_timeout skipped -- Use of the `network' resource not enabled
test_tokenize
test_trace
test_traceback
test_timeout
test_timeout skipped -- Use of the `network' resource not enabled
test_tokenize
test_trace
test_traceback
test_transformer
test_tuple
test_typechecks
test_ucn
test_unary
test_undocumented_details
test_unicode
test_transformer
test_tuple
test_unicode_file
test_unicode_file skipped -- No Unicode filesystem semantics on this platform.
test_unicodedata
test_typechecks
test_ucn
test_unary
test_undocumented_details
test_unicode
test_unicode_file
test_unicode_file skipped -- No Unicode filesystem semantics on this platform.
test_unicodedata
test_univnewlines
test_unpack
test_urllib
test_urllib2
test_urllib2_localnet
test_univnewlines
test_unpack
test_urllib
test_urllib2
test_urllib2_localnet
test_urllib2net
test_urllib2net skipped -- Use of the `network' resource not enabled
test_urllibnet
test_urllibnet skipped -- Use of the `network' resource not enabled
test_urlparse
test_userdict
test_urllib2net
test_urllib2net skipped -- Use of the `network' resource not enabled
test_urllibnet
test_urllibnet skipped -- Use of the `network' resource not enabled
test_urlparse
test_userdict
test_userlist
test_userlist
test_userstring
test_userstring
test_uu
test_uuid
test_uu
test_uuid
    WARNING: uuid.getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
    WARNING: uuid._ifconfig_getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
    WARNING: uuid._unixdll_getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
test test_uuid failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_uuid.py", line 440, in test_uuid4
    equal(u.version, 4)
AssertionError: 6 != 4

    WARNING: uuid.getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
    WARNING: uuid._ifconfig_getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
    WARNING: uuid._unixdll_getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
test test_uuid failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_uuid.py", line 440, in test_uuid4
    equal(u.version, 4)
AssertionError: 6 != 4

test_wait3
test_wait3
test_wait4
test_wait4
test_warnings
test_warnings
test_wave
test_wave
test_weakref
test_weakref
test_whichdb
test_whichdb
test_winreg
test_winreg
test_winreg skipped -- No module named _winreg
test_winreg skipped -- No module named _winreg
test_winsound
test_winsound
test_winsound skipped -- No module named winsound
test_with
test_winsound skipped -- No module named winsound
test_with
test_wsgiref
test_wsgiref
test_xdrlib
test_xdrlib
test_xml_etree
test_xml_etree
test_xml_etree_c
test_xml_etree_c
test_xmllib
test_xmllib
test_xmlrpc
test_xmlrpc
test_xpickle
test_xpickle
test_xrange
test_xrange
test_zipfile
test_zipfile
test_zipfile64
test_zipfile64 skipped -- test requires loads of disk-space bytes and a long time to run
test_zipimport
test_zipimport_support
test_zipfile64
test_zipfile64 skipped -- test requires loads of disk-space bytes and a long time to run
test_zipimport
test_zipimport_support
test_zlib
320 tests OK.
6 tests failed:
    test_ctypes test_distutils test_pep263 test_posix test_sys
    test_uuid
39 tests skipped:
    test_aepack test_al test_applesingle test_bsddb test_bsddb185
    test_bsddb3 test_cd test_cl test_codecmaps_cn test_codecmaps_hk
    test_codecmaps_jp test_codecmaps_kr test_codecmaps_tw test_curses
    test_dl test_epoll test_gl test_imageop test_imgfile test_kqueue
    test_linuxaudiodev test_macos test_macostools test_normalization
    test_ossaudiodev test_pep277 test_py3kwarn test_scriptpackages
    test_smtpnet test_socketserver test_startfile test_sunaudiodev
    test_timeout test_unicode_file test_urllib2net test_urllibnet
    test_winreg test_winsound test_zipfile64
2 skips unexpected on sunos5:
    test_sunaudiodev test_dl
LD_LIBRARY_PATH=/export/home/drlou/illumos-userland-1248/components/python/python26/build/amd64: ./python -E -tt /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/regrtest.py -l 
test_zlib
test_grammar
test_opcodes
test_dict
test_builtin
test_exceptions
test_types
test_unittest
test_doctest
321 tests OK.
7 tests failed:
    test_ctypes test_distutils test_multiprocessing test_pep263
    test_posix test_sys test_uuid
37 tests skipped:
    test_aepack test_al test_applesingle test_bsddb test_bsddb185
    test_bsddb3 test_cd test_cl test_codecmaps_cn test_codecmaps_hk
    test_codecmaps_jp test_codecmaps_kr test_codecmaps_tw test_curses
    test_epoll test_gl test_imgfile test_kqueue test_linuxaudiodev
    test_macos test_macostools test_normalization test_ossaudiodev
    test_pep277 test_py3kwarn test_scriptpackages test_smtpnet
    test_socketserver test_startfile test_sunaudiodev test_timeout
    test_unicode_file test_urllib2net test_urllibnet test_winreg
    test_winsound test_zipfile64
1 skip unexpected on sunos5:
    test_sunaudiodev
LD_LIBRARY_PATH=/export/home/drlou/illumos-userland-1248/components/python/python26/build/i86: ./python -E -tt /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/regrtest.py -l 
[?1034htest_doctest2
test_MimeWriter
test_grammar
test_SimpleHTTPServer
test_opcodes
test_dict
test_StringIO
test___all__
test_builtin
test_exceptions
test___future__
test_types
test__locale
test_unittest
test_abc
test_doctest
test_abstract_numbers
test_aepack
test_aepack skipped -- No module named aepack
test_aifc
test_al
test_al skipped -- No module named al
test_anydbm
[?1034htest_doctest2
test_MimeWriter
test_SimpleHTTPServer
test_applesingle
test_applesingle skipped -- No module named macostools
test_StringIO
test_array
test___all__
test___future__
test__locale
test_abc
test_abstract_numbers
test_aepack
test_aepack skipped -- No module named aepack
test_aifc
test_al
test_al skipped -- No module named al
test_anydbm
test_ast
test_asynchat
test_applesingle
test_applesingle skipped -- No module named macostools
test_array
test_ast
test_asynchat
test_asyncore
test_asyncore
test_atexit
test_audioop
test_augassign
test_base64
test_bastion
test_bigaddrspace
test_bigmem
test_binascii
test_binhex
test_binop
test_bisect
test_bool
test_bsddb
test_bsddb skipped -- No module named _bsddb
test_bsddb185
test_bsddb185 skipped -- No module named bsddb185
test_bsddb3
test_bsddb3 skipped -- Use of the `bsddb' resource not enabled
test_buffer
test_bufio
test_atexit
test_audioop
test_augassign
test_base64
test_bastion
test_bigaddrspace
test_bigmem
test_binascii
test_binhex
test_binop
test_bisect
test_bytes
test_bool
test_bsddb
test_bsddb skipped -- No module named _bsddb
test_bsddb185
test_bsddb185 skipped -- No module named bsddb185
test_bsddb3
test_bsddb3 skipped -- Use of the `bsddb' resource not enabled
test_buffer
test_bufio
test_bz2
test_bytes
test_calendar
test_call
test_capi
test_bz2
test_calendar
test_call
test_capi
test_cd
test_cd skipped -- No module named cd
test_cfgparser
test_cgi
test_charmapcodec
test_cl
test_cl skipped -- No module named cl
test_class
test_cmath
test_cmd
test_cmd_line
test_cd
test_cd skipped -- No module named cd
test_cfgparser
test_cgi
test_charmapcodec
test_cl
test_cl skipped -- No module named cl
test_class
test_cmath
test_cmd_line_script
test_cmd
test_cmd_line
test_code
test_codeccallbacks
test_codecencodings_cn
test_cmd_line_script
test_code
test_codeccallbacks
test_codecencodings_cn
test_codecencodings_hk
test_codecencodings_jp
test_codecencodings_hk
test_codecencodings_jp
test_codecencodings_kr
test_codecencodings_tw
test_codecmaps_cn
test_codecmaps_cn skipped -- Use of the `urlfetch' resource not enabled
test_codecencodings_kr
test_codecmaps_hk
test_codecmaps_hk skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_jp
test_codecmaps_jp skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_kr
test_codecmaps_kr skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_tw
test_codecmaps_tw skipped -- Use of the `urlfetch' resource not enabled
test_codecs
test_codeop
test_coding
test_coercion
test_collections
test_codecencodings_tw
test_codecmaps_cn
test_codecmaps_cn skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_hk
test_codecmaps_hk skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_jp
test_codecmaps_jp skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_kr
test_codecmaps_kr skipped -- Use of the `urlfetch' resource not enabled
test_codecmaps_tw
test_codecmaps_tw skipped -- Use of the `urlfetch' resource not enabled
test_codecs
test_codeop
test_colorsys
test_coding
test_commands
test_coercion
test_compare
test_compile
test_collections
test_compiler
test_colorsys
test_commands
test_compare
test_compile
test_compiler
test_complex
test_complex_args
test_contains
test_contextlib
test_cookie
test_cookielib
test_copy
test_copy_reg
test_cpickle
test_cprofile
test_crypt
test_csv
test_ctypes
test test_ctypes failed -- errors occurred; run in verbose mode for details
test_curses
test_curses skipped -- Use of the `curses' resource not enabled
test_datetime
test_dbm
test_decimal
test_complex
test_complex_args
test_contains
test_contextlib
test_cookie
test_cookielib
test_copy
test_copy_reg
test_cpickle
test_decorators
test_defaultdict
test_deque
test_cprofile
test_crypt
test_csv
test_ctypes
test test_ctypes failed -- errors occurred; run in verbose mode for details
test_curses
test_curses skipped -- Use of the `curses' resource not enabled
test_datetime
test_dbm
test_decimal
test_descr
test_decorators
test_defaultdict
test_deque
test_descrtut
test_difflib
test_dircache
test_descr
test_dis
test_distutils
test_descrtut
test_difflib
test test_distutils failed -- errors occurred; run in verbose mode for details
test_dl
test_dl skipped -- No module named dl
test_docxmlrpc
test_dumbdbm
test_dummy_thread
test_dummy_threading
test_email
test_email_codecs
test_email_renamed
test_enumerate
test_eof
test_epoll
test_epoll skipped -- test works only on Linux 2.6
test_errno
test_exception_variations
test_extcall
test_fcntl
test_file
test_dircache
test_dis
test_distutils
test test_distutils failed -- errors occurred; run in verbose mode for details
test_dl
test_docxmlrpc
test_dumbdbm
test_dummy_thread
test_dummy_threading
test_email
test_email_codecs
test_email_renamed
test_enumerate
test_eof
test_epoll
test_epoll skipped -- test works only on Linux 2.6
test_errno
test_exception_variations
test_extcall
test_fcntl
test_file
test_filecmp
test_fileinput
test_fileio
test_float
test_fnmatch
test_fork1
test_filecmp
test_fileinput
test_fileio
test_float
test_fnmatch
test_fork1
test_format
test_fpformat
test_fractions
test_frozen
test_ftplib
test_funcattrs
test_functools
test_future
test_future3
test_future4
test_future5
test_future_builtins
test_gc
test_gdbm
test_generators
test_genericpath
test_genexps
test_getargs
test_getargs2
test_getopt
test_gettext
test_gl
test_gl skipped -- No module named gl
test_glob
test_global
test_grp
test_gzip
test_hash
test_hashlib
test_heapq
test_format
test_fpformat
test_fractions
test_frozen
test_ftplib
test_hmac
test_hotshot
test_funcattrs
test_functools
test_future
test_htmllib
test_future3
test_htmlparser
test_future4
test_future5
test_future_builtins
test_httplib
test_gc
test_httpservers
test_gdbm
test_generators
test_genericpath
test_genexps
test_getargs
test_getargs2
test_getopt
test_gettext
test_gl
test_gl skipped -- No module named gl
test_glob
test_global
test_grp
test_gzip
test_hash
test_hashlib
test_heapq
test_hmac
test_hotshot
test_htmllib
test_htmlparser
test_httplib
test_httpservers
test_imageop
test_imageop skipped -- No module named imageop
test_imaplib
test_imgfile
test_imgfile skipped -- No module named imgfile
test_imp
test_import
test_importhooks
test_index
test_inspect
test_int
test_int_literal
test_io
test_ioctl
test_isinstance
test_iter
test_iterlen
test_itertools
test_imageop
test_json
test_imaplib
test_imgfile
test_imgfile skipped -- No module named imgfile
test_imp
test_import
test_importhooks
test_index
test_inspect
test_int
test_int_literal
test_io
test_kqueue
test_kqueue skipped -- test works only on BSD
test_largefile
test_lib2to3
test_ioctl
test_isinstance
test_iter
test_iterlen
test_itertools
test_json
test_linecache
test_linuxaudiodev
test_linuxaudiodev skipped -- Use of the `audio' resource not enabled
test_kqueue
test_kqueue skipped -- test works only on BSD
test_list
test_largefile
test_locale
test_lib2to3
test_logging
test_long
test_long_future
test_longexp
test_macos
test_macos skipped -- No module named MacOS
test_macostools
test_macostools skipped -- No module named macostools
test_macpath
test_mailbox
test_linecache
test_linuxaudiodev
test_linuxaudiodev skipped -- Use of the `audio' resource not enabled
test_list
test_locale
test_logging
test_long
test_long_future
test_longexp
test_macos
test_macos skipped -- No module named MacOS
test_macostools
test_macostools skipped -- No module named macostools
test_macpath
test_mailbox
test_marshal
test_math
test_md5
test_memoryio
test_mhlib
test_mimetools
test_mimetypes
test_minidom
test_mmap
test_module
test_modulefinder
test_multibytecodec
test_multibytecodec_support
test_multifile
test_multiprocessing
test_marshal
test_math
test_md5
test_memoryio
test_mhlib
test_mimetools
test_mimetypes
test_minidom
test_mmap
test_module
test_modulefinder
test_multibytecodec
test_multibytecodec_support
test_multifile
test_multiprocessing
test_mutants
test_mutex
test_netrc
test_new
test_nis
test_normalization
test_normalization skipped -- Use of the `urlfetch' resource not enabled
test_ntpath
test_old_mailbox
test_openpty
test_operator
test_optparse
test_os
test_ossaudiodev
test_ossaudiodev skipped -- Use of the `audio' resource not enabled
test_parser
test_peepholer
test_pep247
test_pep263
test test_pep263 crashed -- <type 'exceptions.SyntaxError'>: Non-ASCII character '\xf0' in file /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_pep263.py on line 11, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (test_pep263.py, line 11)
test_pep277
test_pep277 skipped -- test works only on NT+
test_pep292
test_pep352
test_pickle
test_pickletools
test_pipes
test_pkg
test_pkgimport
test_pkgutil
test_platform
test_plistlib
test_poll
test_mutants
test_mutex
test_netrc
test_new
test_nis
test_normalization
test_normalization skipped -- Use of the `urlfetch' resource not enabled
test_ntpath
test_old_mailbox
test_openpty
test_operator
test_optparse
test_os
test_ossaudiodev
test_ossaudiodev skipped -- Use of the `audio' resource not enabled
test_parser
test_peepholer
test_pep247
test_pep263
test test_pep263 crashed -- <type 'exceptions.SyntaxError'>: Non-ASCII character '\xf0' in file /export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_pep263.py on line 11, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details (test_pep263.py, line 11)
test_pep277
test_pep277 skipped -- test works only on NT+
test_pep292
test_pep352
test_pickle
test_pickletools
test_pipes
test_pkg
test_pkgimport
test_pkgutil
test_platform
test_plistlib
test_poll
test_popen
test_popen2
test_poplib
test_posix
test test_posix failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 267, in test_getcwd_long_pathnames
    _create_and_do_getcwd(dirname)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 260, in _create_and_do_getcwd
    os.getcwd()
OSError: [Errno 34] Result too large

test_posixpath
test_pow
test_pprint
test_print
test_profile
test_profilehooks
test_property
test_pstats
test_pty
test_pwd
test_py3kwarn
test_py3kwarn skipped -- test.test_py3kwarn must be run with the -3 flag
test_pyclbr
test_pydoc
test_pyexpat
test_queue
test_popen
test_popen2
test_poplib
test_posix
test test_posix failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 267, in test_getcwd_long_pathnames
    _create_and_do_getcwd(dirname)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 262, in _create_and_do_getcwd
    _create_and_do_getcwd(dirname, current_path_length + len(dirname) + 1)
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_posix.py", line 260, in _create_and_do_getcwd
    os.getcwd()
OSError: [Errno 34] Result too large

test_posixpath
test_pow
test_pprint
test_print
test_profile
test_profilehooks
test_property
test_pstats
test_pty
test_pwd
test_py3kwarn
test_py3kwarn skipped -- test.test_py3kwarn must be run with the -3 flag
test_pyclbr
test_pydoc
test_pyexpat
test_queue
test_quopri
test_random
test_re
test_repr
test_resource
test_rfc822
test_richcmp
test_robotparser
test_runpy
test_sax
test_scope
test_scriptpackages
test_scriptpackages skipped -- No module named aetools
test_select
test_quopri
test_random
test_re
test_repr
test_resource
test_rfc822
test_richcmp
test_robotparser
test_runpy
test_sax
test_scope
test_scriptpackages
test_scriptpackages skipped -- No module named aetools
test_select
test_set
test_sets
test_sgmllib
test_sha
test_shelve
test_shlex
test_shutil
test_signal
test_set
test_sets
test_sgmllib
test_sha
test_shelve
test_shlex
test_shutil
test_signal
test_site
test_slice
test_smtplib
test_smtpnet
test_smtpnet skipped -- Use of the `network' resource not enabled
test_socket
test_site
test_slice
test_smtplib
test_smtpnet
test_smtpnet skipped -- Use of the `network' resource not enabled
test_socket
test_socketserver
test_socketserver skipped -- Use of the `network' resource not enabled
test_softspace
test_sort
test_sqlite
test_ssl
test_startfile
test_startfile skipped -- cannot import name startfile
test_str
test_strftime
test_socketserver
test_socketserver skipped -- Use of the `network' resource not enabled
test_softspace
test_sort
test_sqlite
test_string
test_stringprep
test_strop
test_strptime
test_struct
test_structmembers
test_structseq
test_subprocess
test_ssl
test_startfile
test_startfile skipped -- cannot import name startfile
test_str
test_strftime
test_string
test_stringprep
test_strop
test_strptime
test_struct
test_structmembers
test_structseq
test_subprocess
.
    this bit of output is from a test of stdout in a different process ...
.
    this bit of output is from a test of stdout in a different process ...
test_sunaudiodev
test_sunaudiodev skipped -- no audio device found!
test_sundry
test_symtable
test_syntax
test_sys
test test_sys failed -- errors occurred; run in verbose mode for details
test_tarfile
test_tcl
test_sunaudiodev
test_sunaudiodev skipped -- no audio device found!
test_telnetlib
test_sundry
test_symtable
test_syntax
test_sys
test test_sys failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_sys.py", line 359, in test_43581
    self.assert_(sys.__stdout__.encoding == sys.__stderr__.encoding)
AssertionError

test_tarfile
test_tempfile
test_textwrap
test_thread
test_tcl
test_telnetlib
test_threaded_import
test_threadedtempfile
test_threading
test_tempfile
test_textwrap
test_threading_local
test_thread
test_threaded_import
test_threadedtempfile
test_threadsignals
test_threading
test_time
test_threading_local
test_threadsignals
test_time
test_timeout
test_timeout skipped -- Use of the `network' resource not enabled
test_tokenize
test_trace
test_traceback
test_timeout
test_timeout skipped -- Use of the `network' resource not enabled
test_tokenize
test_trace
test_traceback
test_transformer
test_tuple
test_typechecks
test_ucn
test_unary
test_undocumented_details
test_unicode
test_transformer
test_tuple
test_unicode_file
test_unicode_file skipped -- No Unicode filesystem semantics on this platform.
test_unicodedata
test_typechecks
test_ucn
test_unary
test_undocumented_details
test_unicode
test_unicode_file
test_unicode_file skipped -- No Unicode filesystem semantics on this platform.
test_unicodedata
test_univnewlines
test_unpack
test_urllib
test_urllib2
test_urllib2_localnet
test_univnewlines
test_unpack
test_urllib
test_urllib2
test_urllib2_localnet
test_urllib2net
test_urllib2net skipped -- Use of the `network' resource not enabled
test_urllibnet
test_urllibnet skipped -- Use of the `network' resource not enabled
test_urlparse
test_userdict
test_urllib2net
test_urllib2net skipped -- Use of the `network' resource not enabled
test_urllibnet
test_urllibnet skipped -- Use of the `network' resource not enabled
test_urlparse
test_userdict
test_userlist
test_userstring
test_userlist
test_userstring
test_uu
test_uu
test_uuid
test_uuid
    WARNING: uuid.getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
    WARNING: uuid._ifconfig_getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
    WARNING: uuid._unixdll_getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
test test_uuid failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_uuid.py", line 439, in test_uuid4
    equal(u.variant, uuid.RFC_4122)
AssertionError: 'reserved for future definition' != 'specified in RFC 4122'

    WARNING: uuid.getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
    WARNING: uuid._ifconfig_getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
    WARNING: uuid._unixdll_getnode is unreliable on many platforms.
        It is disabled until the code and/or test can be fixed properly.
test test_uuid failed -- Traceback (most recent call last):
  File "/export/home/drlou/illumos-userland-1248/components/python/python26/Python-2.6.4/Lib/test/test_uuid.py", line 439, in test_uuid4
    equal(u.variant, uuid.RFC_4122)
AssertionError: 'reserved for future definition' != 'specified in RFC 4122'

test_wait3
test_wait3
test_wait4
test_wait4
test_warnings
test_wave
test_warnings
test_weakref
test_wave
test_weakref
test_whichdb
test_whichdb
test_winreg
test_winreg skipped -- No module named _winreg
test_winsound
test_winsound skipped -- No module named winsound
test_with
test_winreg
test_winreg skipped -- No module named _winreg
test_wsgiref
test_winsound
test_winsound skipped -- No module named winsound
test_with
test_xdrlib
test_wsgiref
test_xml_etree
test_xdrlib
test_xml_etree_c
test_xml_etree
test_xmllib
test_xml_etree_c
test_xmlrpc
test_xmllib
test_xpickle
test_xmlrpc
test_xpickle
test_xrange
test_zipfile
test_xrange
test_zipfile
test_zipfile64
test_zipfile64 skipped -- test requires loads of disk-space bytes and a long time to run
test_zipimport
test_zipimport_support
test_zipfile64
test_zipfile64 skipped -- test requires loads of disk-space bytes and a long time to run
test_zipimport
test_zipimport_support
test_zlib
320 tests OK.
6 tests failed:
    test_ctypes test_distutils test_pep263 test_posix test_sys
    test_uuid
39 tests skipped:
    test_aepack test_al test_applesingle test_bsddb test_bsddb185
    test_bsddb3 test_cd test_cl test_codecmaps_cn test_codecmaps_hk
    test_codecmaps_jp test_codecmaps_kr test_codecmaps_tw test_curses
    test_dl test_epoll test_gl test_imageop test_imgfile test_kqueue
    test_linuxaudiodev test_macos test_macostools test_normalization
    test_ossaudiodev test_pep277 test_py3kwarn test_scriptpackages
    test_smtpnet test_socketserver test_startfile test_sunaudiodev
    test_timeout test_unicode_file test_urllib2net test_urllibnet
    test_winreg test_winsound test_zipfile64
2 skips unexpected on sunos5:
    test_sunaudiodev test_dl
test_zlib
make[1]: Leaving directory `/export/home/drlou/illumos-userland-1248/components/python/python26/build/amd64'
322 tests OK.
6 tests failed:
    test_ctypes test_distutils test_pep263 test_posix test_sys
    test_uuid
37 tests skipped:
    test_aepack test_al test_applesingle test_bsddb test_bsddb185
    test_bsddb3 test_cd test_cl test_codecmaps_cn test_codecmaps_hk
    test_codecmaps_jp test_codecmaps_kr test_codecmaps_tw test_curses
    test_epoll test_gl test_imgfile test_kqueue test_linuxaudiodev
    test_macos test_macostools test_normalization test_ossaudiodev
    test_pep277 test_py3kwarn test_scriptpackages test_smtpnet
    test_socketserver test_startfile test_sunaudiodev test_timeout
    test_unicode_file test_urllib2net test_urllibnet test_winreg
    test_winsound test_zipfile64
1 skip unexpected on sunos5:
    test_sunaudiodev
make[1]: Leaving directory `/export/home/drlou/illumos-userland-1248/components/python/python26/build/i86'
